source/simulator/parameters.cc

```diff
@@ -767,6 +767,17 @@ namespace aspect
                          "\n\n"
                          "Note that while more than one operation can be selected it only makes sense to "
                          "pick one rotational and one translational operation.");
+
+      prm.declare_entry("Remove rotational nullspace in linear solver", "false",
```

Comment:
Shouldnt the default be `auto` to avoid users having to set this parameter for all gmg models?

```suggestion
      prm.declare_entry("Remove rotational nullspace in linear solver", "auto",
```